skillserver/skillserver.go

```diff
@@ -199,16 +206,10 @@ func verifyJSON(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
 
 // Run all mandatory Amazon security checks on the request.
 func validateRequest(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
-	// Check for debug bypass flag
-	devFlag := r.URL.Query().Get("_dev")
-
-	isDev := devFlag != ""
 
-	if !isDev {
-		isRequestValid := IsValidAlexaRequest(w, r)
-		if !isRequestValid {
-			return
-		}
+	isRequestValid := IsValidAlexaRequest(w, r)
+	if !isRequestValid {
```

Comment:
we can inline this unless we use isRequestValid elsewhere. Ie:

```go
if !IsValidAlexaRequest {
    return
}
```

We might also want to log if the request isn't valid.